opennms-doc/guide-admin/src/asciidoc/text/events/event-configuration.adoc

```diff
@@ -123,9 +138,23 @@ Not all events will have values for all tokens, and some refer specifically to i
     The operator instructions for the event.
 `%mouseovertext%`::
     The mouse over text for the event.
+`%tticketid%`::
+    The trouble ticket id associated with the event, if available.
 `%primaryinterface%`::
 The primary interface IP address for the node given in `%nodeid%` if available.
 
+.Asset tokens
+A node may have additional asset meta data stored for it. You can access that metadata using the `asset` replacement token, which takes the form:
```

Comment:
```suggestion
A node may have additional asset records stored for it.
You can access these records using the `asset` replacement token, which takes the form:
```

asset meta-data -> asset records to help avoid being confuse with the new "node meta-data"
one sentence per line